models/model.js

```diff
@@ -44,3 +44,15 @@ exports.fetchCommentsByArticleId = (article_id) => {
     })
 }
 
+exports.insertComment = (article_id, commentData) => {
+    const currentDate = new Date(Date.now());
+    const dataArray = [commentData.body, article_id, commentData.username, 0, currentDate]
```

Comment:
I believe that the way the comments table was seeded means that you don't need to insert a date 😄